docs/go/tools.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Go tools
+nav_order: 300
+parent: Go Services
+permalink: /go/tools
+---
+## BitBox Base: Go tools
+
+We have some Go tools that are part of this repository:
```

Comment:
Same here. How about:
> Several custom Go tools are part of this repository:

Also: further detailed documentation of these tools in `/docs/go/tools/`?

Reply:
> Same here. How about:
> 
> > Several custom Go tools are part of this repository:
I changed the text to `Several Go tools are part of this repository`, thanks.
> 
> Also: further detailed documentation of these tools in `/docs/go/tools/`?

Not sure what this is referring to? We're discussing on the `docs/go/tools.md` document; what text would you like me to add?